src/main/kotlin/io/github/bbortt/buessle/app/configuration/AspectConfiguration.kt

```diff
@@ -0,0 +1,14 @@
+package io.github.bbortt.buessle.app.configuration
+
+import io.github.bbortt.buessle.app.aspect.ApiLoggingAspect
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class AspectConfiguration {
+
+    @Bean
+    fun apiLoggingAspect(): ApiLoggingAspect {
+        return ApiLoggingAspect();
```

Comment:
Do we need this? Couldn't you just annotate the `ApiLogginAspect` as a `@Component`?

Reply:
Hm yes, valid point. I am not a great fan of components as they are "kind of hidden", but if we keep them all in `aspect` package we're good to go. I'll commit it later.